scrapper.py

```diff
@@ -25,39 +35,107 @@ class Crawler:
     """
     Crawler implementation
     """
-    def __init__(self, seed_urls, max_articles: int):
-        pass
+
+    def __init__(self, seed_urls, total_max_articles: int):
+        self.seed_urls = seed_urls
+        self.total_max_articles = total_max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        article_summaries = article_bs.find("div", class_="obj_article_summary")
+        articles = article_summaries.find("div", class_="title")
+        for link in articles:
```

Comment:
where do we control the number of articles we are going to collect?

Reply:
still actual

Reply:
I control it in def _extract_url(self, article_bs), where list of retrieved URLs shouldn't be longer than the integer number "total_max_articles"